### PR TITLE
Update Documentation for Nested Event Properties [ING-2385] [ING-2261]

### PR DESCRIPTION
**Pull request by toksfifo**
# Pull Request/Issue Resolution

**Description of Change:**
Update documentation for nested event properties

Closes [ING-2385](https://jira.braze.com/browse/ING-2385) and [ING-2261](https://jira.braze.com/browse/ING-2261)

### Is this change associated with a Braze feature/product release?
- [x] Yes (Date is still TBD)
- [ ] No

---
---

<details>
<summary>PR Checklist</summary>

- [x] Check that all links work!
- [x] Ensure you have completed [our CLA](https://www.braze.com/docs/cla/).
- [x] Tag @Timothy-Kim and @KellieHawks as a reviewer when your work is _done and ready to be reviewed for merge_.
- [x] Tag others as Reviewers as necessary.
- [x] If you have modified any links, be sure to add redirects to `config/nginx.conf.erb`.

</details>

<details>
<summary>Writing & Style Quick Tips</summary>

- Use the formatting shown on the [Styling Test Page](https://github.com/Appboy/braze-docs/blob/develop/_docs/_home/styling_test_page.md) to create unique templates and vehicles for your information.
- Use Action verbs!
- Capitalize proper nouns, unique Braze tools and features, custom Braze channels, and all Header titles.
- Redirect when necessary!
- Always speak to the Braze customer - refer to them as “you”, whether they are an Engineer/Developer, a Marketer, or an Admin.
- On Voice: You should aim to be a Friendly Professor. Imagine, if you will, the smartest, but also kindest professor you had. You know, the one who showed you your potential and tested you, but also went slowly in class so you could take great notes and learn some things? Don't be too peppy, but don't be formal. You’re not selling anything - you’re instructing and informing and enabling.

_Consult the [Docs Text Formatting Guide](https://github.com/Appboy/braze-docs/wiki/Special-Formatting) and the [Docs Writing Style Guide & Best Practices](https://github.com/Appboy/braze-docs/wiki/Writing-Style-Guide-&-Best-Practices) for more details._

</details>

<details>
<summary>ATTENTION: For PR Reviewers</summary>

- [ ] Read our [Reviewing a PR page](https://github.com/Appboy/braze-docs/wiki/Reviewing-a-PR) for more on our reviewing suggestions.
- [ ] Preview all changes in the linked Heroku environment (click `View deployment` button below, then "Docs". A `502` error just means you should refresh until you see the Docs Home page.
</details>

---
---

### CODE REVIEW

**Review by mkigikm — APPROVED** [review]

File: _docs/_user_guide/data_and_analytics/custom_data/nested_object_support.md [hunk 1]
```diff
@@ -71,50 +74,66 @@ properties:{
...
 
-`r_details.name` - "Mcdonalds"<br>
-`r_details.location.city` - "Montclair"
+`r_details.name`: "Mcdonalds"<br>
+`r_details.location.city`: "Montclair"
 {% endtab %}
 {% endtabs %}
 
+{% alert note %} Triggering is not yet supported for In-App Messages.  {% endalert %}
+{% alert note %} If your event property contains the `[]` or `.` characters, escape them by wrapping the chunk in double-quotes. For instance, `"songs[].album".yearReleased` will match an event with the literal property `"songs[].album"`.  {% endalert %}
```
Comment (johndinh) [inline _docs/_user_guide/data_and_analytics/custom_data/nested_object_support.md:124]: great call out

**Comment by johndinh** [comment]
LGTM

**Comment by KellieHawks** [comment]
@toksfifo Just checking in to see if you'd like this merged.

**Comment by toksfifo** [comment]
> @toksfifo Just checking in to see if you'd like this merged.

Thanks for checking in. Should be another week or so (@johndinh what do you think?).

**Comment by johndinh** [comment]
I think we're ready for this. We'll need these docs before we can feature flip on for customers.

**Review by KellieHawks — APPROVED** [review]

**Pull request merged**